typescript/packages/common-charm/src/space-manager.ts

```diff
@@ -42,18 +42,18 @@ export const processSchema: JSONSchema = {
   },
 } as const;
 
-export class CharmManager {
+export class SpaceManager {
   private space: Space;
   private charmsDoc: DocImpl<DocLink[]>;
   private charms: Cell<Cell<Charm>[]>;
 
-  constructor(private spaceId: string) {
-    this.space = getSpace(this.spaceId);
+  constructor(private spaceURI: string) {
```

Comment:
is this always a URI? or will it still be an arbitrary string sometimes?

maybe deserves a doc comment

Reply:
arbitrary for now, but will be a public key